src/item.cpp

```diff
@@ -6003,7 +6003,14 @@ std::string item::tname( unsigned int quantity, bool with_prefix, unsigned int t
     }
 
     if( has_var( "NANOFAB_ITEM_ID" ) ) {
-        tagtext += string_format( " (%s)", nname( itype_id( get_var( "NANOFAB_ITEM_ID" ) ) ) );
+        if( has_flag( flag_NANOFAB_TEMPLATE_SINGLE_USE ) ) {
+            //~ Single-use descriptor for nanofab templates. %s = name of resulting item. The leading space is intentional.
+            tagtext += string_format( _( " (SINGLE USE %s)" ), nname( itype_id( get_var( "NANOFAB_ITEM_ID" ) ) ) );
+            tagtext += string_format( _( " (SINGLE USE %s)" ),
+                                      nname( itype_id( get_var( "NANOFAB_ITEM_ID" ) ) ) );
```

Comment:
```suggestion
            tagtext += string_format( _( " (SINGLE USE %s)" ),
                                      nname( itype_id( get_var( "NANOFAB_ITEM_ID" ) ) ) );
```
How did that get there? Last update (hopefully)

Reply:
Don't worry about it I coulda just hit it with the style tomorrow when I checked on it and I appreciate the translator note it's something I always forget. If this one fails just lemme astyle it in VS.